pagerduty/event_orchestration_cache_variable.go

```diff
@@ -0,0 +1,119 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+)
+
+type EventOrchestrationCacheVariableService service
+
+type EventOrchestrationCacheVariableCondition struct {
+	// A PCL string: https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview
+	Expression string `json:"expression,omitempty"`
+}
+
+// Configuration for a cache variable changes depending on the type:
+//   - if `Type` is `recent_value`; then use `Regex` and `Source`
+//   - if `Type` is `trigger_event_count`; then use `TTLSeconds`
+type EventOrchestrationCacheVariableConfiguration struct {
+	Type       string `json:"type,omitempty"`
+	Regex      string `json:"regex,omitempty"`
+	Source     string `json:"source,omitempty"`
+	TTLSeconds int    `json:"ttl_seconds,omitempty"`
+}
+
+// A reference to a related object (e.g. an User, etc)
+type EventOrchestrationCacheVariableReference struct {
+	ID   string `json:"id,omitempty"`
+	Type string `json:"type,omitempty"`
+	Self string `json:"self,omitempty"`
+}
+
+type EventOrchestrationCacheVariable struct {
+	ID            string                                   			`json:"id,omitempty"`
+	Name          string                                    		`json:"name,omitempty"`
+	Disabled      bool 																          `json:"disabled"`
+	Conditions    []*EventOrchestrationCacheVariableCondition   `json:"conditions"`
+	Configuration *EventOrchestrationCacheVariableConfiguration `json:"configuration,omitempty"`
+	CreatedAt     string                           							`json:"created_at,omitempty"`
+	CreatedBy     *EventOrchestrationCacheVariableReference 		`json:"created_by,omitempty"`
+	UpdatedAt     string                           							`json:"updated_at,omitempty"`
+	UpdatedBy     *EventOrchestrationCacheVariableReference 		`json:"updated_by,omitempty"`
+}
+
+type EventOrchestrationCacheVariablePayload struct {
+	CacheVariable *EventOrchestrationCacheVariable `json:"cache_variable,omitempty"`
+}
+
+type ListEventOrchestrationCacheVariablesResponse struct {
+	Total          int                                `json:"total,omitempty"`
+	CacheVariables []*EventOrchestrationCacheVariable `json:"cache_variables,omitempty"`
+}
+
+const CacheVariableTypeGlobal string = "global"
+const CacheVariableTypeService string = "service"
+
+func buildEventOrchestrationCacheVariableUrl(cacheVariableType string, orchestrationId string, cacheVariableId string) string {
+	if cacheVariableType == CacheVariableTypeService {
+		return fmt.Sprintf("%s/services/%s/cache_variables/%s", eventOrchestrationBaseUrl, orchestrationId, cacheVariableId)
+	}
+
+	return fmt.Sprintf("%s/%s/cache_variables/%s", eventOrchestrationBaseUrl, orchestrationId, cacheVariableId)
+}
+
+func (s *EventOrchestrationCacheVariableService) ListContext(ctx context.Context, cacheVariableType string, orchestrationId string) (*ListEventOrchestrationCacheVariablesResponse, *Response, error) {
```

Comment:
Please remove the `Context` suffix from methods, it is no longer needed. This apply to the rest of methods too. Thank you!

Reply:
Addressed in https://github.com/heimweh/go-pagerduty/pull/149/commits/cb1dc594997dc4cea3275e5ee1a1f5e2993627be